src/components/forms/FamilyForm.tsx

```diff
@@ -922,56 +890,18 @@ export const FamilyForm = ({ family: loadedFamily }: TProps) => {
                   </Button>
                 </Box>
               )}
-              <Box position='relative' padding='10'>
-                <Divider />
-                <AbsoluteCenter bg='gray.50' px='4'>
-                  Events
-                </AbsoluteCenter>
-              </Box>
-              {!loadedFamily && (
-                <Text>
-                  Please create the family first before attempting to add events
-                </Text>
-              )}
-              {familyEvents.length && (
-                <Flex direction='column' gap={4}>
-                  {familyEvents.map((familyEvent) => (
-                    <FamilyEvent
-                      canModify={canModify(orgName, isSuperUser, userAccess)}
-                      eventId={familyEvent}
-                      key={familyEvent}
-                      onEditClick={(event) => onEditEntityClick('event', event)}
-                      onDeleteClick={onEventDeleteClick}
-                    />
-                  ))}
-                </Flex>
-              )}
-              {loadedFamily && (
-                <Box>
-                  <Button
-                    isDisabled={
-                      !canModify(
-                        loadedFamily?.organisation,
-                        isSuperUser,
-                        userAccess,
-                      )
-                    }
-                    onClick={() => onAddNewEntityClick('event')}
-                    rightIcon={
-                      familyEvents.length === 0 ? (
-                        <WarningIcon
-                          color='red.500'
-                          data-test-id='warning-icon-event'
-                        />
-                      ) : undefined
-                    }
-                  >
-                    Add new Event
-                  </Button>
-                </Box>
-              )}
+              <FamilyEventList
+                familyEvents={familyEvents}
+                canModify={canModify}
```

Comment:
From Slack - suggest we pass in the result of calling `canModify` rather than the method itself. Might also trim down the number of props we need to pass into the child component

So rather than:
`canModify={canModify}`
Make it:
`canModify={canModify(...arguments)}`

Reply:
Changes implemented